packages/api-explorer/src/ApiExplorer.tsx

```diff
@@ -57,11 +57,11 @@ export interface SpecItem {
 export type SpecItems = KeyedCollection<SpecItem>
 
 export interface ApiExplorerProps {
-  runItCallback?: RunItCallback
   specs: SpecItems
+  sdk?: Looker31SDK | Looker40SDK
```

Comment:
is `Looker31SDK` needed if `RunIt` is only available for 4.0 and above?

Reply:
Please ignore my previous question. As I reviewed the PR it became obvious that this will be needed in the extension version. The 4.0 and above requirement for `RunIt` is only for the standalone version.